main.go

```diff
@@ -563,6 +643,16 @@ func (e *Exporter) parseStats(ch chan<- prometheus.Metric, stats map[net.Addr]me
 		"moves_to_cold":     e.itemsMovesToCold,
 		"moves_to_warm":     e.itemsMovesToWarm,
 		"moves_within_lru":  e.itemsMovesWithinLru,
+		"number_hot":        e.itemsNumberHot,
+		"number_warm":       e.itemsNumberWarm,
+		"number_cold":       e.itemsNumberCold,
+		"number_temp":       e.itemsNumberTemp,
+		"age_hot":           e.itemsAgeOldesHot,
+		"age_warm":          e.itemsAgeOldesWarm,
```

Comment:
It looks like this `itemsMetrics` list assumes that everything is a counter. But some of these new metrics are gauges. I guess we need to fix this.

Reply:
Yes definitely, didn't realize it.. will try to fix!